bindings/matrix-sdk-ffi/src/widget.rs

```diff
@@ -0,0 +1,154 @@
+use std::sync::Arc;
+
+use matrix_sdk::async_trait;
+
+use crate::room::Room;
+
+#[derive(uniffi::Record)]
+pub struct Widget {
+    /// Information about the widget.
+    pub info: WidgetInfo,
+    /// Communication channels with a widget.
+    pub comm: Arc<WidgetComm>,
```

Comment:
I would really prefer `information` and `communication`. We don't use short names anywhere else in the SDK. Thoughts?

Reply:
I think this structure is going to change going forward anyways, can we consider this later?

Reply:
OK.